messages/pt.json

```diff
@@ -1,4 +1,5 @@
 {
+  "admin/store-name.description": "This is your store's name, in Portuguese",
```

Comment:
Is that expected? 🤔 

```suggestion
  "admin/store-name.description": "This is your store's name, in Portuguese",
```

Reply:
Whaaaaat?? I don't know where this message came from 🤔

Reply:
Removed it everywhere